src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue

```diff
@@ -232,6 +236,20 @@ export class GenericContactPropertyEditor extends mixins(Lang) {
     return !isEmpty(this.subscriberPropertyFields)
   }
 
+  get subscriberPropertyFieldsForSelector(): IContactPropertyOptionForUISelector[] {
+    return map(this.subscriberPropertyFields, (field: IContactPropertyOption) => {
+      let shouldDisplay
```

Comment:
is `shouldDisplay` correct here? We display them, just disabled.

Reply:
Thanks @jcbashdown , you're right, will rename it to `shouldDisable`